common-lib/src/main/scala/com/gu/mediaservice/lib/elasticsearch/ElasticSearchClient.scala

```diff
@@ -24,7 +24,7 @@ trait ElasticSearchClient {
   private lazy val settings: Settings =
     ImmutableSettings.settingsBuilder
       .put("cluster.name", cluster)
-      .put("client.transport.sniff", true)
+      .put("client.transport.sniff", false)
```

Comment:
Is this something that should only be applied when we are updating mappings?

Reply:
It doesn't look like it's affecting the other scripts.